Project.toml

```diff
@@ -50,6 +51,7 @@ StaticArrays = "0.12"
 StatsBase = "0.32, 0.33"
 Tables = "1.0"
 Transducers = "0.4"
+ReferenceFrameRotations = "0.5"
```

Comment:
Amazing. I was just considering ReferenceFrameRotations.jl today. The package is well-written and well-documented, so we shouldn't have problems adding it as a dependency :+1:

Reply:
Nice. It's a really nice library for rotations